package.json

```diff
@@ -49,31 +49,46 @@
   },
   "license": "MIT",
   "private": true,
+  "workspaces": [
+    "demo",
+    "packages/*"
+  ],
   "packageManager": "yarn@1.22.19",
   "engines": {
     "vscode": ">=1.0.0"
   },
+  "main": "out/language-client.js",
   "keywords": [
     "mdx",
     "Markdown",
     "Markdown React"
   ],
   "scripts": {
+    "build": "esbuild @mdx-js/language-client @mdx-js/language-server --bundle --platform=node --target=node16 --external:vscode --external:typescript --outdir=out --sourcemap=inline",
     "lint": "eslint . --cache -f friendly",
-    "prepare": "patch-package && simple-git-hooks && yarn-deduplicate --strategy fewer || exit 0"
+    "prepare": "patch-package && simple-git-hooks && yarn-deduplicate --strategy fewer || exit 0",
+    "start": "yarn workspace @mdx-js/monaco-demo run start",
+    "vscode:prepublish": "yarn build --minify"
   },
   "devDependencies": {
-    "@1stg/common-config": "^7.2.0",
-    "@changesets/changelog-github": "^0.4.7",
-    "@changesets/cli": "^2.25.2",
+    "@1stg/common-config": "^7.1.1",
+    "@changesets/changelog-github": "^0.4.6",
+    "@changesets/cli": "^2.24.4",
+    "@types/react": "^18.0.20",
+    "esbuild": "^0.15.0",
```

Comment:
Could these version ranges be widened?
For example:
```diff
-    "@1stg/common-config": "^7.1.1",
+    "@1stg/common-config": "^7.0.0",
```

Reply:
This must have happened due to merge conflicts. I’ll resolve it.